config-helpers.go

```diff
@@ -48,7 +49,7 @@ func bucketKeyMappings(bucketKeyMapString string) map[string]string {
 		bucketKeyMap[bucketKeyArray[0]]=bucketKeyArray[1]
 	}
 	
-	fmt.Println(bucketKeyMap)
+	log.Debug(bucketKeyMap)
```

Comment:
same with the keyMapString, provide a short description as to what's logged.  Also, if the string is already logged, logging the map is kind of redundant. Probably better just to log the  final map